plugins/disassemble/disassemble_main.ml

```diff
@@ -487,19 +492,22 @@ let save_knowledge ~had_knowledge ~update digest = function
     Knowledge.save (Toplevel.current ()) path
   | Some _ -> ()
 
-let create_and_process input outputs passes loader target update
+let create_and_process input link outputs passes loader target update
     kb print_missing ctxt =
   let uses_file_loader = Sys.file_exists loader &&
                          Fn.non Filename.is_implicit loader in
+  let link = List.dedup_and_sort link ~compare:String.compare in
```

Comment:
We already are doing this in project.ml

Reply:
I was doing it here so that, no matter the order (or duplicates), it would lead to the same digest. But I see now that semantically the order matters so it would make sense to remove it.